script/formatting/formatter.py

```diff
@@ -56,81 +56,82 @@
 
 header_comment_1 = header_comment_line_1 + header_comment_line_2
```

Comment:
What should we do to format this section correctly? I was thinking of merging these individual strings to a complete whole. This will eliminate all these unwanted global variables as well.

Reply:
Feel free to find find a better way to put this into a string. But make sure one is still able to see the output somehow, so that it can be changed easily.